libs/dh/core/shell/src/lib/notification-center/dh-notifications-center.component.ts

```diff
@@ -91,4 +133,30 @@ export class DhNotificationsCenterComponent {
       overlayY: 'top',
     },
   ];
+
+  showNotificationBanner(): void {
+    this.hotToast.show(DhNotificationBannerComponent, {
+      position: 'top-right',
+      dismissible: true,
+      autoClose: false,
+      style: {
+        'background-color': 'rgba(219, 219, 219, 0.3)',
+        border: '1px solid #bdbdbd', // gray-400
+        width: '345px',
+        color: 'rgba(0, 0, 0, 0.87)', // on-light-high-emphasis
```

Comment:
Comments: Seems abit od to provide styles like this, is there an other way?

Reply:
Found out that most of the properties can be moved to a CSS variable. Those that couldn't are kept here. Any existing Watt colors are now taken from a service instead

Reply:
Perfect